Update Exchange-Teams-interact.md (#7939)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/Exchange-Teams-interact.md

<sup>9</sup> A minimum of an Exchange Online Plan 1 license is also required. For more information, see [Search for Teams chat data for on-premises users](/microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users).

<sup>10</sup> On-premises users can use Teams to update their profile picture even if the `SetPhotoEnabled` Outlook on the Web mailbox policy is set to `false`.

> <sup>11</sup> Time-zone preference is not supported in a on-prem mailbox scenario

Reviewer: ```suggestion

<sup>11</sup> Time-zone preference is not supported in an on-premises mailbox scenario.
```
